packages/kit/src/runtime/app/navigation.js

```diff
@@ -102,7 +102,7 @@ export const preloadCode = /* @__PURE__ */ client_method('preload_code');
 export const beforeNavigate = /* @__PURE__ */ client_method('before_navigate');
 
 /**
- * A lifecycle function that runs the supplied `callback` immediately before we navigate to a new URL.
+ * A lifecycle function that runs the supplied `callback` immediately before we navigate to a new URL, except during full-page navigations.
```

Comment:
It seems odd to me that we'd mention it only on this one and not on the other lifecycle functions 
```suggestion
 * A lifecycle function that runs the supplied `callback` immediately before we navigate to a new URL except during full-page navigations.
```

Reply:
`before/afterNavigate` mention that it occurs `we navigate to a new URL`, which I read as including full-page navigations. Happy to clarify it by mentioning that they run on full-page navigations?